### PR TITLE
BOM-1679

### PR DESCRIPTION
**Pull request by awais786**
https://openedx.atlassian.net/browse/BOM-1679

dropped 2.7 support.
Python3.5 support.
make upgrade.

Boto is working fine in 3.5 so no need to update the code.

### CODE REVIEW

**Comment by awais786** [comment]
@morenol Please review.

File: gen_cert.py [hunk 1]
```diff
@@ -577,7 +579,7 @@ def _generate_v1_certificate(
         styleOpenSans.alignment = TA_LEFT
 
         paragraph_string = u"<b><i>{0}: {1}</i></b>".format(
```
Comment (morenol) [inline gen_cert.py:581]: I think that `u` is redundant here

File: gen_cert.py [hunk 2]
```diff
@@ -858,9 +860,9 @@ def _generate_v2_certificate(
 
         paragraph_string = u"{0}: {1}".format(self.course, self.long_course)
```
Comment (morenol) [inline gen_cert.py:861]: same comment as above

**Comment by awais786** [comment]
> I just have a comment that is repeated in a file several times but I think that we can merge without that, on the other hand, why aren't you adding python 3.8 to the travis matrix? @awais786
I was planning to add in new PR after merging this.

**Comment by awais786** [comment]
@morenol tests not running and build is getting green.

**Pull request merged**